.github/SUPPORT.md

```diff
@@ -30,13 +31,18 @@ get support for using HPX.
 ### Mailing Lists/Groups
 
 * [hpx-users group](mailto:hpx-users@stellar-group.org), see
-  [here](https://www.mail-archive.com/hpx-users@stellar.cct.lsu.edu) for past
-  conversations.
+  [here](https://www.mail-archive.com/hpx-users@stellar-group.org) for past
+  conversations. This is the main mailing list for user questions and HPX
+  announcements.
+* [hpx-devel group](mailto:hpx-devel@stellar-group.org), see
+  [here](https://www.mail-archive.com/hpx-devel@stellar-group.org) for past
+  conversations. This is the main mailing list for development discussion
+  and voting.
```

Comment:
Should we also mention the PMC mailing list?

Reply:
Yes, good idea.